zomes/likes/code/src/lib.rs

```diff
@@ -0,0 +1,184 @@
+extern crate hdk;
+mod hdk_membrane;
+
+
+// Following @sphinxc0re's suggestion for a capabilities macro
+// that also produces the JSON for the whole capability
+// including all functions.
+capabilities!{
```

Comment:
Yes! I like `capabilities` better

Reply:
The plural is confusing to me. Here we are describing just one Capability as the comment suggest, no?
I am also worried about namespace for the macros, and suggest prefixing here also: `zome_capability!`

Reply:
On a general note, trying to figure out specs with just one code example seems pretty confusing...

Reply:
No, there are actually two capabilities defined here. It's a bit hard to see with all those comments.. ;) there is `main (public)` below.

Reply:
Ok! So can you change the comment from `whole capability` to `whole zome` ?